spec/lookup_spec.rb

```diff
@@ -17,8 +15,8 @@
     describe "#initialize(table_name)" do
```

Comment:
@samfrench I would look to modify this (and any other tests you notice) so that their `describe` doesn't include the specific signature. Because as you can see, in this case the signature has changed and the spec is reading incorrectly. I'd prefer to just have it as `describe "#initialize"` so that it can adapt as the code requirements change.